input/pagecontent/terminology.md

```diff
@@ -66,6 +67,7 @@ The list below shows the code systems used in the value sets above.
 |[NameUse](https://hl7.org/fhir/R4/codesystem-name-use.html)|[NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)|FHIR|
 |[RequestIntent](https://hl7.org/fhir/R4/codesystem-request-intent.html)|[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/codesystem-request-status.html)|[AU eRequesting RequestStatus](ValueSet-au-erequesting-request-status.html)|FHIR|
+|[SexParameterForClinicalUseCS](http://terminology.hl7.org/CodeSystem/sex-parameter-for-clinical-use)|[SexParameterForClinicalUse](http://terminology.hl7.org/ValueSet/sex-parameter-for-clinical-use)|THO|
 |[SNOMED CT (Australian extension)](https://www.healthterminologies.gov.au/access-clinical-terminology/access-fhir-terminology-resources/code-systems/?ui:filter=snomed)|[Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1), [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1),[RANZCR Radiology Referral](ValueSet-ranzcr-radiology-referral.html)|NCTS|
 |[Tags for the Identification of Languages](https://terminology.hl7.org/CodeSystem-v3-ietf3066.html)|[Common Languages in Australia](https://healthterminologies.gov.au/fhir/ValueSet/common-languages-australia-2)|FHIR|
```

Comment:
Commenting here to get an inline comment. 
Sex Parameter for Clinical Use value set also includes codes ('unknown') from the Data Absent Reason code system. 
Please update the Data Absent Reason code system entry in the table to add Sex Parameter for Clinical Use value set to the list of value sets. I suggest changing the Data Absent Reason URL to be the canonical and changing the Published from 'FHIR' to 'HL7 Terminology'.

Reply:
done